uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/list/QuotesListItemView.kt

```diff
@@ -69,10 +69,10 @@ class QuotesListItemView @JvmOverloads constructor(
         startLoading()
         quoteNameText.text = vehicleDetails.fleet.name
 
-        var logoImage = VehicleMappingsProvider.getVehicleMappings()?.let {
+        val logoImageUrl = VehicleMappingsProvider.getVehicleMappings()?.let {
             vehicleDetails.vehicle.getCorrespondingLogoMapping(it)?.vehicleImagePNG
         } ?: vehicleDetails.fleet.logoUrl
-        loadImage(logoImage)
+        loadImages(logoImageUrl , vehicleDetails.fleet.logoUrl)
```

Comment:
If `logoImageUrl` defaults to `vehicleDetails.fleet.logoUrl`, why are we passing it separately as well?

Reply:
It's just a failsafe in case the vehicleMappings are empty. There shouldn't be any scenario in which this should happen